frontend/src/pages/projects/screens/detail/notebooks/useNotebookDeploymentSize.ts

```diff
@@ -31,12 +29,10 @@ const useNotebookDeploymentSize = (
   if (!size) {
     return {
       size: null,
-      error:
-        'Workbench size is currently unavailable. Check your dashboard configuration to view size information.',
     };
   }
 
-  return { size, error: '' };
+  return { size };
```

Comment:
If `requests` differ but `limits` match a predefined size, that size is shown and the UI displays the incorrect `requests` in the UI.

eg:
```
          resources:
            limits:
              cpu: '2'
              memory: 8Gi
            requests:
              cpu: '2'
              memory: 9Gi
```
![image](https://github.com/user-attachments/assets/1a3168da-0b00-406d-9b7f-636e7d9529e9)
![image](https://github.com/user-attachments/assets/f3ba50fd-e251-42b6-97cf-dc26507272eb)


Expected to see `custom` and the correct values in the UI.


As a result of this, when editing this notebook, the size is selected as `small` and even if I do not change the size, I lose my custom values and it changes to the small container size values.